### DISCUSSION TITLE
Add year to calendar tile

### DISCUSSION THREAD

**Discussion by ljpaul** [indiana-university/rivet-source#625] [Feature and enhancement requests]
It would be great to be able to optionally include a year in the calendar tile.

[Example can be found on this page.](https://zion.sitehost-test.iu.edu/)

**Comment by levimcg** [comment]
@ljpaul Thanks for the feedback!

**Comment by levimcg** [comment]
@ljpaul Just a heads up. An optional year element was added in #636 and was released in [version 2.1.0](https://github.com/indiana-university/rivet-source/releases/tag/v2.1.0). I'm going to go ahead and mark this as answered. 👍🏻
